src/i_video.c

```diff
@@ -731,10 +731,16 @@ static void I_ResetTargetRefresh(void);
  #define I_CpuPause()
 #endif
 
+#ifdef _MSC_VER
+#pragma optimize("s", on) // Don't unroll the wait loop
```

Comment:
Could this probably be the single most important change in this PR?

Reply:
I had the same question but no, this doesn't supersede the other changes in this PR. Testing the various permutations of the three de-optimizations, I still need all three.